CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ the code was deployed.
 - Production deployment instructions in the README.
 - Created string_convert namespace for string_convert.cpp functions.
 
+### Added
```

Comment:
I see you have a conflict in this file that will need to be resolved before you're able to merge. This is normal, we almost always have conflicts in the CHANGELOG when there is more than one PR open at once. When you resolve this conflict, please make sure that this line describing your change ends up in the ## [unreleased] section and not under ## [6.4.0]. Thanks!

Reply:
I fixed the conflict in the changelog in my latest commit